libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm

```diff
@@ -1,263 +1,263 @@
+EK2_MAGE_P_NSE	0.005
+EK2_MAG_CAL	3
+EK2_RNG_USE_HGT	-1
+EK2_MAG_LRN_LIM 30
+FENCE_ENABLE	0
+FLTMODE1	0
+FLTMODE2	0
+FLTMODE3	0
+FLTMODE4	5
+FLTMODE5	2
+FLTMODE6	9
+FRAME_CLASS	1
+FRAME_TYPE	1
+FS_GCS_ENABLE	0
+FS_THR_ENABLE	6
+FS_THR_VALUE	0
+GND_EFFECT_COMP	1
+GPS_HDOP_GOOD	175
+GPS_POS1_X	0.182
+GPS_POS1_Y	-0.05
+GPS_POS1_Z	-0.04
+GPS_SAVE_CFG	2
+GPS_TYPE	2
+INS_ACCEL_FILTER	15
+INS_FAST_SAMPLE	7
+INS_GYRO_FILTER	20
+INS_GYR_CAL	0
+INS_POS1_X	-0.077
+INS_POS1_Y	0
+INS_POS1_Z	-0.05
+INS_POS2_X	-0.077
+INS_POS2_Y	0
+INS_POS2_Z	-0.05
+INS_POS3_X	-0.077
+INS_POS3_Y	0
+INS_POS3_Z	-0.05
+INS_TRIM_OPTION	0
+INS_USE	1
+INS_USE2	1
+INS_USE3	1
+INS_HNTCH_ENABLE 1
+INS_HNTCH_FREQ 56
+INS_HNTCH_BW   37
+INS_HNTCH_ATT  40
+INS_HNTCH_REF  0.193
+INS_HNTCH_MODE  1
+INS_HNTCH_HMNCS 15
+LAND_REPOSITION	0
+LAND_SPEED	40
+LAND_SPEED_HIGH	200
+LOG_DISARMED	0
+LOG_FILE_BUFSIZE	60
+LOG_FILE_DSRMROT 1
+LOG_REPLAY	0
+MIS_RESTART	0
+MOT_BAT_CURR_MAX	60
+MOT_BAT_CURR_TC	2
+MOT_BAT_POW_MAX	2000
+MOT_BAT_VOLT_MAX	50.8
+MOT_BAT_VOLT_MIN	35
+MOT_HOVER_LEARN	0
+MOT_PWM_MAX	1940
+MOT_PWM_MIN	1060
+MOT_SAFE_DISARM	0
+MOT_SPIN_ARM	0.11
+MOT_SPIN_MAX	1
+MOT_SPIN_MIN	0.15
+MOT_THST_EXPO	0.8
+PILOT_THR_BHV	0
+PILOT_TKOFF_ALT	150
+PLND_ACC_P_NSE	2
+PLND_CAM_POS_X	0.165
+PLND_CAM_POS_Y	-0.014
+PLND_CAM_POS_Z	0.151
+PLND_ENABLED	1
+PLND_LAND_OFS_X	0
+PLND_LAND_OFS_Y	-2.6
+PLND_TYPE	2
+PLND_YAW_ALIGN	18000
+PSC_ACC_XY_FILT	3.5
+RC1_DZ	30
+RC1_MAX	2000
+RC1_MIN	1002
+RC1_TRIM	1500
+RC2_DZ	30
+RC2_MAX	1999
+RC2_MIN	1000
+RC2_TRIM	1500
+RC3_DZ	30
+RC3_MAX	1998
+RC3_MIN	999
+RC3_TRIM	1500
+RC4_DZ	40
+RC4_MAX	1997
+RC4_MIN	1002
+RC4_TRIM	1500
+RCMAP_PITCH	2
+RCMAP_ROLL	1
+RCMAP_THROTTLE	3
+RCMAP_YAW	4
+RNGFND1_TYPE	7
+RNGFND2_ADDR	112
+RNGFND2_MAX_CM	10000
+RNGFND2_MIN_CM	50
+RNGFND2_TYPE	0
+RNGFND1_ADDR	102
+RNGFND1_GNDCLEAR	15
+RNGFND1_MAX_CM	9000
+RNGFND1_MIN_CM	0
+RNGFND1_POS_X	-0.124
+RNGFND1_POS_Y	-0.097
+RNGFND1_POS_Z	0.037
+RNGFND1_SCALING	1
+RNGFND1_STOP_PIN	55
+SERIAL0_BAUD	115
+SERIAL0_PROTOCOL	2
+SERIAL1_BAUD	921
+SERIAL1_PROTOCOL	1
+SERIAL2_BAUD	57
+SERIAL2_PROTOCOL	5
+SERIAL3_BAUD	38
+SERIAL3_PROTOCOL	5
+SERIAL4_BAUD	115
+SERIAL4_PROTOCOL	100
+SERIAL5_BAUD	57
+SERIAL5_PROTOCOL -1
+SERVO1_FUNCTION	33
+SERVO2_FUNCTION	34
+SERVO3_FUNCTION	36
+SERVO4_FUNCTION	35
+SYSID_MYGCS	255
+TERRAIN_ENABLE	0
+WPNAV_ACCEL	120
+WPNAV_ACCEL_Z	75
+WPNAV_RADIUS	200
+WPNAV_SPEED	1600
+WPNAV_SPEED_DN	400
+WPNAV_SPEED_UP	400
+COMPASS_ORIENT	35
+MOT_THST_HOVER	0.185
+ATC_RAT_PIT_I	0.1
+FS_EKF_ACTION	4
+COMPASS_PRIMARY	1
+FS_EKF_THRESH	0.8
+COMPASS_MOT2_X	1.89
+COMPASS_MOT2_Y	0.37
+COMPASS_MOT2_Z	-1.2
+COMPASS_ORIENT2	36
+EK2_ALT_SOURCE	2
+GPS_AUTO_SWITCH	1
+GPS_POS2_X	0.182
+GPS_POS2_Y	0.05
+GPS_POS2_Z	-0.04
+GPS_TYPE2	2
+GPS_DELAY_MS2   100
+INS_LOG_BAT_MASK 0
+BATT_AMP_PERVLT	45
+BATT_LOW_VOLT	36
+BATT_FS_LOW_ACT	6
+BATT_LOW_MAH	0
+LOIT_ANG_MAX	20
+PSC_ACCZ_D	0
+PSC_ACCZ_FF	0
+PSC_ACCZ_FLTE	15
+PSC_ACCZ_I	0.5
+PSC_ACCZ_IMAX	800
+PSC_ACCZ_P	0.2
+PSC_ANGLE_MAX	0
+PSC_POSXY_P	1.35
```

Comment:
@AkshayMatternet Could you please set the defaults to:
`PSC_POSXY_P 0.75
PSC_VELXY_I 0.75
PSC_VELXY_P 1.5
`

Reply:
Yes I can, I was planning to add those params as a separate PR, to make it easy for comparison, between 3.6.7 and 4.0

Reply:
Thanks. That's fine

Reply:
Added the changes, and are up for review. Thanks @zalice